test/cpp/end2end/xds/BUILD

```diff
@@ -64,18 +71,26 @@ grpc_cc_library(
         "//src/proto/grpc/testing:echo_messages_proto",
         "//src/proto/grpc/testing:echo_proto",
         "//src/proto/grpc/testing/duplicate:echo_duplicate_proto",
-        "//src/proto/grpc/testing/xds:ads_for_test_proto",
-        "//src/proto/grpc/testing/xds:lrs_for_test_proto",
-        "//src/proto/grpc/testing/xds/v3:ads_proto",
-        "//src/proto/grpc/testing/xds/v3:cluster_proto",
-        "//src/proto/grpc/testing/xds/v3:discovery_proto",
-        "//src/proto/grpc/testing/xds/v3:endpoint_proto",
-        "//src/proto/grpc/testing/xds/v3:http_connection_manager_proto",
-        "//src/proto/grpc/testing/xds/v3:http_filter_rbac_proto",
-        "//src/proto/grpc/testing/xds/v3:listener_proto",
-        "//src/proto/grpc/testing/xds/v3:lrs_proto",
-        "//src/proto/grpc/testing/xds/v3:route_proto",
-        "//src/proto/grpc/testing/xds/v3:router_proto",
+        "//src/proto/grpc/lookup/v1:rls_config_proto",
```

Comment:
Can we also use the rls protos from the grpc/grpc-proto repo instead of having our own local copy?

Feel free to do this in a separate PR.

Reply:
Jan and I had some discussion about using the grpc/grpc-proto protos. The benefits are: 1. (as mentioned) remove the copy; 2. remove the hacks we did to override the proto package path (e.g., health.proto supposed to be located at grpc/health/v1/health.proto, but we put it under src/protos).

That's probably a bigger issue. It's on my todo list.